web/src/features/map/Map.tsx

```diff
@@ -84,14 +89,19 @@ export default function MapPage(): ReactElement {
         'fill-color': '#FFFFFF',
         'fill-opacity': ['case', ['boolean', ['feature-state', 'hover'], false], 0.3, 0],
       } as mapboxgl.FillPaint,
+      statesBorder: {
+        'line-color': theme.stateBorderColor,
+        'line-width': 1.4,
+        'line-opacity': 0.9,
+        'line-dasharray': [1, 1],
+      } as mapboxgl.LinePaint,
```

Comment:
Not sure if this comment is in the right place but would it be possible to change the border color between light and dark mode?